_posts/2018-07-25-microplatforms-product-plaforms-as-code.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title:  "Microplatforms - Product Platforms as Code"
+date:   2018-07-25 12:00:00 +0000
```

Comment:
I am not sure you actually need the date, as it's part of the filename. Therefore you could simplify.

Reply:
I'm not a Jekyll expert, but as I understand it, this value is needed when the `_site` directory structure is being built

Reply:
On the redirects - good point.
I'll look into that tomorrow

Reply:
I meant is that I think that Jekyll can infer the post date from the file name, therefore there is maybe no need to repeat it in the YAML.

Reply:
So I've just tested the way Jekyll builds the site both with and without the `date` field in the front matter.

It all depends on the desired degree of ordering precision...

If you only care about resolving a person's blogs to the nearest day, then you only need the date in blog filename; however, if someone creates two blog posts on the same day, then if present, the time part of the datestamp in the `date` field will be used to order posts to a resolution of one second.  If the `date` field is missing, then the order of blogs created on the same day does not seem to be defined.

Since in the `CONTRIBUTING.md` file, I've already described adding the `date` field to the front matter and its purpose, I think it would be easier just to leave it as it is